src/layouts/Tutorial.tsx

```diff
@@ -1 +1,284 @@
-export const TutorialLayout = ({ children }) => <div>{children}</div>
+// Libraries
+import {
+  Badge,
+  Box,
+  chakra,
+  Divider,
+  Flex,
+  HeadingProps,
+  Kbd,
+  Text,
+  TextProps,
+  useToken,
+} from "@chakra-ui/react"
+import { useRouter } from "next/router"
+import dynamic from 'next/dynamic'
+ 
+
+
+// Components
+import { ButtonLink } from "@/components/Buttons"
+import CallToContribute from "@/components/CallToContribute"
+import Card from "@/components/Card"
+import Codeblock from "@/components/Codeblock"
+// TODO: Implement CrowdinContributors after intl is implemented
+// import CrowdinConbtirbutors from "@/components/FileContributorsCrowdin"
+import Emoji from "@/components/Emoji"
+import EnvWarningBanner from "@/components/EnvWarningBanner"
+const FeedbackCard = dynamic(() => import("@/components/FeedbackCard"), { ssr: false })
```

Comment:
Why do we need to import it this way? Could this be related to the error in the build?

Reply:
I'm suspecting this may have been related to an improperly guarded use of `window` inside FeedbackCard (ie, `window && surveyUrl &&...` should be `typeof window !== "undefined" && surveyUrl...`)